lab/lab1/part5-project-euler.html

```diff
@@ -35,6 +35,18 @@
 
     ===================== */
 
+    /*Problem:
+    If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9.
+    The sum of these multiples is 23. Find the sum of all the multiples of 3 or 5 below 1000.*/
+
+    var sum = 0;
+    for(var i = 1; i < 1000; i+=1) {
+      if(i%3==0 || i%5==0){
+        sum+=i
+      }
+    }
+    console.log(sum)
```

Comment:
Nice. Here's a writeup on optimizing this problem: https://adamdrake.com/an-unreasonably-deep-dive-into-project-euler-problem-1.html